### DISCUSSION TITLE
Invalid config on development environment

### DISCUSSION THREAD

**Discussion by Poeschl** [UnchartedBull/OctoDash#1395] [Q&A] [answered]
Hi, 
I tried to setup the development environment for integrating 3 new themes as counterparts to my octoprint themes.
Everything went fine for install and setup according to your contributing file. The electron browser is starting, but gives me then the error of an invalid config file.

Is there a specific location to put the `config.json`? I already tried inside the app folder or inside the assets as ``config.json` and `config.testing.json`. Or did I miss something else?

**Comment by UnchartedBull** [comment, marked as answer]
There was an error with a library that got upgraded. Fixed with https://github.com/UnchartedBull/OctoDash/commit/bd92653609d3d97b2a372bb82a155c7bff9307dd. Please pull the latest changes and everything should be working again. Feel free to create PRs for your themes if you like :)

**Reply by Poeschl** [reply]
Thank you. 😉 I will, for sure 😁
